crates/infra/cli/src/utils.rs

```diff
@@ -60,6 +62,10 @@ impl Terminal {
         const DEFAULT_WIDTH: usize = 100;
         const BANNER_GLYPHS: usize = 6; // "╾┤  ├╼"
 
```

Comment:
that would also prevent this from running in CI, which is really helpful in reading large logs.

Instead, what do you think about writing it to stderr (unconditionally)? It is a standard/conventional location for progress reporting.